static/src/inline-svgs/icon/instagram.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
You probably want to run this through SVGO, and then remove the xmlns attribute which SVGO doesn't remove